sdk/core/core-auth/review/core-auth.api.md

```diff
@@ -19,6 +19,13 @@ export class AzureKeyCredential implements KeyCredential {
     update(newKey: string): void;
 }
 
+// @public
+export class AzureSasCredential implements SasCredential {
+    constructor(signature: string);
+    get signature(): string;
```

Comment:
Are we only using a getter here because we wanted a simpler contract for `SasCredential` ? Feels a little weird to me.

I'd prefer to change the contract to `getSignature(): string` if we really think this needs to be a method body and not a simple property.

Reply:
@xirzec Keeping a simpler contract for `KeyCredential` was the logic we used when using an analogous getter in `AzureKeyCredential`. It allows us to keep the signature for the interface to be a simple property, so a user can optionally just pass in `{ key }` or `{ signature }` to the client constructor and not have to pass it as `{ getSignature: () => signature }`.

Reply:
I agree with @witemple-msft here and I think we are all on board with how things look now, so I am going to resolve this.  Let me know if I misunderstand your position here, Jeff.